parsl/dataflow/task_status_poller.py

```diff
@@ -35,6 +35,20 @@ def status(self) -> Dict[object, JobStatus]:
     def executor(self) -> ParslExecutor:
         return self._executor
 
+    def scale_in(self, n):
+        ids = self._executor.scale_in(n)
+        if ids is not None:
```

Comment:
`executor.scale_in` should return a list of ids that were removed, it shouldn't return `None`. If it failed to remove the blocks it is supposed to return an empty list.